mopidy/audio/actor.py

```diff
@@ -503,8 +504,8 @@ def on_start(self) -> None:
             self._setup_playbin()
             self._setup_outputs()
             self._setup_audio_sink()
-        except GLib.Error as exc:
-            logger.exception(exc)
+        except GLib.Error:
+            logger.exception("Unknown GLib error on audio startup.")
```

Comment:
Wouldn't this be less unknown with the exception message logged like previously?